### ISSUE TITLE
Gitlab enableReleasePackages() missing / not implemented for themes

### ISSUE THREAD

**Issue by b0rt** [YahnisElsts/plugin-update-checker#467]
## Error Description
Hey there, 

in the readme is statet [here](https://github.com/YahnisElsts/plugin-update-checker#how-to-release-a-gitlab-update)

under 'GitLab Releases using Generic Packages:'
that one could set ` $myUpdateChecker->getVcsApi()->enableReleasePackages();` to enable the lookup for release packages..

but when using as described an error is returned
```
Fatal error: Uncaught Error: Call to undefined method Puc_v4p11_Vcs_GitLabApi::enableReleasePackages() in /var/www/html/wp-content/themes/theme-name/functions.php:103 S
```

How can one use the ReleasePackege functionality ?

I did see preparations for the feature in https://github.com/YahnisElsts/plugin-update-checker/blob/3a65e7ff3e7cf7a6f4e87ce41260d1c3321c5333/Puc/v4p11/Vcs/GitLabApi.php#L392

## Context
functions.php
```
$myUpdateChecker = new Puc_v4p11_Vcs_ThemeUpdateChecker(
    new Puc_v4p11_Vcs_GitLabApi( 'https://gitlab.my-company.com/user/theme/' )
);
$myUpdateChecker->setAuthentication( 'randomstringwithnumb3rs' );
$myUpdateChecker->getVcsApi()->enableReleasePackages();
```

everything works fine when using 'master' but i would like to use ther packages because the repository contains a lot of not needed files

**Comment by YahnisElsts** [comment]
That feature is currently only available in the `master` branch. It's not included in the latest release (4.11). It will be included in the next release, whenever that happens.

So if you want to use `enableReleasePackages()`, you'll need to use the `master` branch, and maybe tweak the class names to avoid conflict with existing version 4.11 copies used by other plugins.

**Comment by b0rt** [comment]
ah i see .. the mentioned function in the readme is part of this [commit](https://github.com/YahnisElsts/plugin-update-checker/commit/a5dd2bb951345966d6432d4e0b54cdf844e0e82e) from September 2021 and is not yet 'released' and i was using the release version

;) you have been faster.. i was just typing this.. thank you kindly :)
